backend/ventserver/protocols/server.py

```diff
@@ -1,6 +1,8 @@
 """Sans-I/O ventilator backend server protocol."""
 
 from typing import Optional, Union, Tuple
+import subprocess
```

Comment:
The `subprocess` module should not be used by any protocol. In fact, it is almost like an I/O operation. I wonder if it would be reasonable to make an I/O endpoint which sole responsibility is to kill a process (or just run an arbitrary shell command) when a method is called? Then the responsibility of the `Server` or `Backend` protocol (probably the receive filter) is to only keep track of the time since the last received websocket message and activate a flag in `ReceiveOutputEvent` once the websocket has been unresponsive for too long. And there would be some code in `integration._trio` which makes the I/O endpoint run the command to kill the process when the flag is activated.

Reply:
Also, I believe Trio has its own functions for spawning subprocesses in an asynchronous manner: https://trio.readthedocs.io/en/stable/reference-io.html#spawning-subprocesses

Reply:
Even I was hesitant about adding a `subprocess` in protocol.  No, I don't think that it would be reasonable to make an I/O endpoint just for spawning a kill process, since we don't need to interact with the process. I was think of something similar to `process_clock`, an async process whose sole job is to check frontend and spawn frontend kill process.

Reply:
That sounds reasonable. Let's put that function in the `io` subpackage rather than the `integration` subpackage.